### PR TITLE
Adding stashcache-origin-01.ultralight.org into CIT infrastructure

### PR DESCRIPTION
**Pull request by andres-moya**
Hi, 

I am doing it my first time. Please check i am doing it right. 

Not sure multiple Secondary allowed in ContactLists

### CODE REVIEW

File: topology/California Institute of Technology/Caltech CMS Tier2/CIT_CMS_T2.yaml [hunk 1]
```diff
@@ -257,6 +257,46 @@ Resources:
...
+          Name: Justas Balcas
+        Secondary:
+          ID: 5e2a78ef254d5b584751f4de9cd490287301e8e7
+          Name: Andres Moya
+        Tertiary:
+          ID: 2b89ad55977c20b374ea58f9a9de452aef3c7bc1
+          Name: Joosep Pata
+    FQDN: stashcache-origin-01.ultralight.org
+    ID: 1009
+    Service: XRootD origin server
```
Comment (matyasselmeci) [inline topology/California Institute of Technology/Caltech CMS Tier2/CIT_CMS_T2.yaml]:
```suggestion
    Service:
      XRootD origin server:
```

File: topology/California Institute of Technology/Caltech CMS Tier2/CIT_CMS_T2.yaml [hunk 2]
```diff
@@ -257,6 +257,46 @@ Resources:
...
+      Description: StashCache origin server for user data
```
Comment (matyasselmeci) [inline topology/California Institute of Technology/Caltech CMS Tier2/CIT_CMS_T2.yaml]:
```suggestion
        Description: StashCache origin server for user data
```

**Comment by matyasselmeci** [comment]
Also, do you have a DN for this resource?

**Comment by andres-moya** [comment]
> 
> 
> Also, do you have a DN for this resource?

FQDN? stashcache-origin-01.ultralight.org?

 Subject: DC=org, DC=incommon, C=US, ST=CA, L=Pasadena, O=California Institute of Technology, OU=High Energy Physics, CN=stashcache-origin-01.ultralight.org

**Comment by andres-moya** [comment]
found more buttons to click :) seems all submitted from my side?

**Comment by andres-moya** [comment]
Uff, this time pretty much everything done. "1 change requested ".  Merging can be performed automatically once the requested changes are addressed. i think it was done? no?

**Comment by andres-moya** [comment]
still can't run a service:

https://topology.opensciencegrid.org/stashcache/origin-authfile?fqdn=stashcache-origin-01.ultralight.org

returns:

# Error generating authfile for this FQDN: No working StashCache resource/VO combinations found
# Please check configuration in OSG topology or contact help@opensciencegrid.org

odd?

**Pull request merged**